docs/tutorial/window-customization.md

```diff
@@ -1,42 +1,58 @@
 # Window Customization
 
-The `BrowserWindow` module is the foundation of your Electron application, and it exposes
-many APIs that can change the look and behavior of your browser windows. In this
-tutorial, we will be going over the various use-cases for window customization on
-macOS, Windows, and Linux.
+The [`BrowserWindow`][] module is the foundation of your Electron application, and it exposes many APIs that let you customize the look and behavior of your app’s windows. This tutorial covers how to implement various use-cases for window customization on macOS, Windows, and Linux.
 
-## Create frameless windows
+## Custom Titlebars
 
-A frameless window is a window that has no [chrome][]. Not to be confused with the Google
-Chrome browser, window _chrome_ refers to the parts of the window (e.g. toolbars, controls)
-that are not a part of the web page.
+### Basic Tutorial
 
-To create a frameless window, you need to set `frame` to `false` in the `BrowserWindow`
-constructor.
+Application windows have a default [chrome][] applied by the OS. Not to be confused with the Google Chrome browser, window _chrome_ refers to the parts of the window (e.g. titlebar, toolbars, controls) that are not a part of the main web content. While the default titlebar provided by the OS chrome is sufficent for simple usecases, many applications opt to remove it. Implementing a custom titlebar can help your application feel more modern and consistent across platforms.
```

Comment:
The previous iteration of this doc used "title bar", and I think that's how I see it spelled across the Windows and macOS documentation. It's also how we spell out the API in camelCase (titleBarStyle). Can we revert the spelling?